src/one_dragon_qt/app/directory_picker.py

```
@@ -238,62 +284,71 @@ def _on_browse_clicked(self):
+        ok, key, pos, ch = validate_install_path(text)
+        if ok:
+            self.path_status_label.setText(self.translator.get_text('path_ok'))
+            self.path_status_label.setStyleSheet('color: #00a854;')
+            self.confirm_btn.setEnabled(True)
+        else:
+            self.path_status_label.setText(self.translator.get_text(key, pos=pos, ch=ch))
+            self.path_status_label.setStyleSheet('color: #d13438;')
+            self.confirm_btn.setEnabled(False)
+
+    def _validate_and_apply_path(self, path: str) -> bool:
+        """校验路径合法性并应用：非法字符走状态栏提示（不再弹窗），仅保留非空目录确认弹窗。通过返回 True。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)